SwiftTweaks/TweakAction.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  TweakAction.swift
+//  SwiftTweaks
+//
+//  Created by Jarosław Pendowski on 28/04/2017.
+//  Copyright © 2017 Khan Academy. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public typealias TweakActionClosure = () -> Void
+
+public class TweakAction {
+	public enum Error: Swift.Error {
+		case wrongIdentifier
+	}
+	
+	public typealias ClosureIdentifier = UInt
+	
+	private var lastToken: ClosureIdentifier = 0
+	private var closures: [ClosureIdentifier: TweakActionClosure] = [:]
```

Comment:
This should maybe hold a weak reference to the callbacks

Reply:
Ah, I can't add a `weak` reference here because the compiler only allows weak references for `class` types!